client/src/components/framework/layoutSkeleton.tsx

```diff
@@ -5,7 +5,7 @@ import React from "react";
 /* App dependencies */
 import Controls from "common/components/Controls/Controls";
 import Layout from "./layout";
-import LeftSidebarSkeleton from "../leftSidebar/leftSidebarSkeleton";
+import LeftSidebarSkeleton from "../LeftSidebar/LeftSidebarSkeleton";
```

Comment:
I am going to leave LeftSidebarSkeleton where it is for now cause I don't want to conflict with whatever choices you made about RightSidebarSkeleton.